check-cla/action.yml

```diff
@@ -102,6 +102,34 @@ runs:
               name: '${{ inputs.label }}'
             })
 
+    # checkout conda/infrastructure to update .clabot
+    - uses: actions/checkout@v3
+      if: steps.contributors.outputs.hasSigned == 'false'
+      with:
+        repository: conda/infrastructure
+
+    # update .clabot
+    - shell: python
+      if: steps.contributors.outputs.hasSigned == 'false'
+      run: |
+        import json
+        from pathlib import Path
+
+        path = Path(".clabot")
+        clabot = json.loads(path.read_text())
+        clabot["contributors"].append("${{ github.actor }}")
+        path.write_text(json.dumps(clabot))
+
+    # create PR
+    - uses: peter-evans/create-pull-request@v4
+      id: cla-pr
+      if: steps.contributors.outputs.hasSigned == 'false'
+      with:
+        branch: cla-${{ github.actor }}
+        commit-message: Adding CLA singee ${{ github.actor }}
+        title: Adding CLA singee ${{ github.actor }}
+        body: Adding CLA signee @${{ github.actor }}
```

Comment:
Should we `@` the user here? They'll get notified etc.

Reply:
The @ is fine and keeps the author updated regarding the current state of verification. 

Would be nice to also add the PR URL from where the action was triggered to the body here. That shows where the contributor works on and where to retrigger the cla bot after the merge.